src/main/java/org/folio/circulation/domain/notice/NoticeEventType.java

```diff
@@ -7,7 +7,7 @@
 
   CHECK_IN("Check in"),
   CHECK_OUT("Check out"),
-  RENEW("Renew"),
+  RENEWED("Renewed"),
```

Comment:
This is only an observation, it won't block the pull request.

From a domain perspective this is interesting and peeked my curiosity. This event type is named in the past tense (which I strongly prefer, especially for user initiated events), yet the others are not.

Can you help me understand why the distinction for renewed?

(I've tried raising this modelling aspect a few times, for example [UICIRC-200](https://issues.folio.org/browse/UICIRC-200?focusedCommentId=45304&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-45304), [UICIRC-198](https://issues.folio.org/browse/UICIRC-198?focusedCommentId=45513&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-45513) with limited interest, so I'm just curious)

Reply:
I'm not really sure why this event naming is different from others. On UI this option is named "Renew":
![image](https://user-images.githubusercontent.com/42538529/60430845-fe32f500-9c06-11e9-8ce1-cf9cb2f4f39f.png)
I just fixed value so that it matches storage schema.
And I agree, I prefer past tense naming for events too.